Libraries/Modal/Modal.js

```diff
@@ -58,6 +59,7 @@ class Modal extends React.Component {
     return (
       <RCTModalHostView
         animated={this.props.animated}
```

Comment:
Also `console.warn` would be nice to have here, e.g. "animated={true} prop is deprecated, use animationType="slide" instead"